src/utils/api/index.test.js

```diff
@@ -0,0 +1,109 @@
+import {
```

Comment:
Nice :clap: 

FYI we decided not to write tests on api methods since these are just wrappers over `request`, so testing only the utils code is spot on :ok_hand: